File: metronic_rails_v8.2.3/rails/metronic_rails_v8.2.3_demo1/rails_demo1/starterkit/app/controllers/auth_controller.rb
class AuthController < ApplicationController

  def initialize
    super
    # Include the main layout bootstrap file
    require_relative "../views/#{Rails.configuration.settings.KT_THEME_LAYOUT_DIR}/_bootstrap/auth"

    # Initialize the main layout bootstrap class
    KTBootstrapDefault.new.init(helpers)
  end

  def signin
    helpers.addJavascriptFile('custom/authentication/sign-in/general.js')

    render "pages/auth/signin"
  end

  def signup
    helpers.addJavascriptFile('custom/authentication/sign-up/general.js')

    render "pages/auth/signup"
  end

  def reset_password
    helpers.addJavascriptFile('custom/authentication/reset-password/reset-password.js')

    render "pages/auth/reset-password"
  end

  def new_password
    helpers.addJavascriptFile('custom/authentication/reset-password/new-password.js')

    render "pages/auth/new-password"
  end
end

File: metronic_symfony_v8.2.3/symfony/metronic_symfony_v8.2.3/starterkit/src/Controller/AuthController.php
<?php

namespace App\Controller;

use App\Controller\Bootstrap\DefaultLayoutController;
use Symfony\Component\HttpFoundation\Response;

class AuthController extends DefaultLayoutController
{
    public function signin(): Response
    {
        $this->theme->addJavascriptFile('js/custom/authentication/sign-in/general.js');

        return $this->render('pages/auth/signin.html.twig');
    }

    public function signup(): Response
    {
        $this->theme->addJavascriptFile('js/custom/authentication/sign-up/general.js');

        return $this->render('pages/auth/signup.html.twig');
    }

    public function reset_password(): Response
    {
        $this->theme->addJavascriptFile('js/custom/authentication/reset-password/reset-password.js');

        return $this->render('pages/auth/reset-password.html.twig');
    }

    public function new_password(): Response
    {
        $this->theme->addJavascriptFile('js/custom/authentication/reset-password/new-password.js');

        return $this->render('pages/auth/new-password.html.twig');
    }
}